service/api/types.go

```diff
@@ -59,8 +59,11 @@ type Breakpoint struct {
 	// Breakpoint condition
 	Cond string
 
-	// tracepoint flag
+	// Tracepoint flag, signifying this is a tracepoint.
 	Tracepoint bool `json:"continue"`
+	// TraceReturn will set a breakpoint on all possible return
```

Comment:
This makes it sound like setting this flag will automatically create a bunch of breakpoints at all return addresses, but it just flags a single breakpoint.

Reply:
Doc leftover from a previous implementation idea, will change.